walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +362,62 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void optionalWithGaps() {
+    // Have one person, but make it so that there is not enough room at any point in the day to
+    // have the meeting.
```

Comment:
Please update the first part of the comment.

Reply:
Updated the comment